app/controllers/telephone_appointments_controller.rb

```diff
@@ -34,7 +34,11 @@ def create_step_2
     if request.xhr?
       render partial: 'times', locals: { times: @times }
     else
-      telephone_appointment.advance! { render :new }
+      if telephone_appointment.start_at.nil?
```

Comment:
Can we flip this logic around and say `if telephone_appointment.start_at?` instead?